### DISCUSSION TITLE
Fixed corvus name

### DISCUSSION THREAD

**Discussion by Nishant9947** [ROM-builders/temporary#15028] [General]
**Corvus-R is now **Corvus-AOSP****

**Link :-** https://github.com/Corvus-AOSP/android_manifest/tree/12-test

_That's why every corvus builder is facing test error._

**Proven links :-** https://cirrus-ci.com/task/6100158155849728
https://cirrus-ci.com/task/6428955199668224

**Merge this pull request :-** _pull/15056_

https://cirrus-ci.com/build/4728591287058432

@Apon77

**Comment by Apon77** [comment]
They are still using 12-test branch, testing branch should not be added

**Reply by Megh-Rana** [reply]
12-test branch was just bad naming, we used 12-test and continued till the end, now the new branches are in Corvus-AOSP with 12 and 13 as names

Ig it can now be changed 

@Apon77
